build.sh

```diff
@@ -280,9 +281,11 @@ if [[ ! "$FIRST_USER_NAME" =~ ^[a-z][-a-z0-9_]*$ ]]; then
 	exit 1
 fi
 
-if [[ -n "${APT_PROXY}" ]] && ! curl --silent "${APT_PROXY}" >/dev/null ; then
-	echo "Could not reach APT_PROXY server: ${APT_PROXY}"
-	exit 1
+if [[ -n "${APT_PROXY}" && ! proxy_check ]]; then
```

Comment:
`proxy_check` already has `[[ -n "${APT_PROXY}" ]]`?

Reply:
You are right. Early implementations of `proxy_check` were not checking the environment and I forgot to update `build.sh`, probably. Since it is not breaking things, didn't notice it during tests.

Reply:
You are right. Early implementations of `proxy_check` were not checking the environment and I forgot to update `build.sh`, probably. Since it is not breaking things, didn't notice it during tests.